homeassistant/components/lupusec/__init__.py

```diff
@@ -73,10 +75,50 @@ def setup(hass: HomeAssistant, config: ConfigType) -> bool:
     return True
 
 
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up this integration using YAML is not supported."""
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up this integration using UI."""
+
+    if hass.data.get(DOMAIN) is None:
+        hass.data.setdefault(DOMAIN, {})
```

Comment:
The whole idea of `.setdefault` is to set it to `{}` only when it doesn't exist ;)

Reply:
Removed if statement.